docs/public/api/GettingStarted.rst

```diff
@@ -0,0 +1,14 @@
+Getting Started
```

Comment:
lets move all of this to /sdk/.

There is not much to document in this repo for the API, as the APIs for all 3 signals are coming from .NET runtime repo.